developerportal/templates/organisms/event-speakers.html

```diff
@@ -13,16 +13,18 @@ <h2>Speakers</h2>
       {% endwith %}
     {% endfor %}
   </div>
-  <div class="event-speakers-more">
-    <div class="event-speakers-actions">
-      <button class="mzp-c-button js-toggle" data-controls="#speaker-list" data-hide>See all</button>
+  {% if speakers|list|slice:"6:" %}
+    <div class="event-speakers-more">
+      <div class="event-speakers-actions">
+        <button class="mzp-c-button js-toggle" data-controls="#speaker-list" data-hide>See all</button>
+      </div>
+      <div class="card-container card-container-people" id="speaker-list" hidden>
+        {% for block in speakers|list|slice:"6:" %}
```

Comment:
Nitpick, but I’d be tempted to cache this as a variable above since it’s being computed twice